content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -34,6 +34,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
   - has a new property for visits, `ended_at`.
   - has new properties: `followed`, `online`, `group_ids`, `state`.
 - The [**Form field**](/messaging/agent-chat-api/v3.3/#form-fields) event's `type` property has a new possible value, `subject`. It's a new type that can appear in ticket forms.
+- Added new event [**Form**](/messaging/agent-chat-api/v3.3/#form)
```

Comment:
I'm guessing @ale-kacperczyk will notice you used past tense in here, while the other entries use present tense